src/Tribe/Google/Maps_API_Key.php

```diff
@@ -47,14 +47,15 @@ public function filter_tribe_addons_tab_fields( array $addon_fields ) {
 
 			'gmaps-js-api-info-box' => array(
 				'type' => 'html',
-				'html' => '<p>' . esc_html__( 'You can optionally specify a Google Maps JavaScript API key The Events Calendar will use to make requests. This is strongly recommended to avoid reaching daily query limits.',
-						'the-events-calendar' ) . '</p>',
+				'html' => '<p>' . sprintf( __( 'We highly recommend that you specify a valid %s for The Events Calendar to use. Doing this will help prevent problem with maps, especially for sites that receive a lot of traffic.',
```

Comment:
I know this is the wording specified in the ticket ... but shouldn't it be _'prevent problems'_ rather than _'prevent problem'?_